DontPanicLabs.Ifx.Configuration.Local/README.md

```diff
@@ -1 +1,95 @@
-# DontPanicLabs.Ifx.Configuration.Local
\ No newline at end of file
+# Local Configuration
+This project supports three configuration sources:
+ - Environment variables
+ - The `appsettings.json` file
+ - The .NET User Secrets store
+
+## Environment Variables
+The Config class first checks the `appsettings.json` file for the presence of the `skipEnvironmentVariables` entry. If this entry is found, and it is set to true, environment variables will not be loaded.
+```json
+{
+    "skipEnvironmentVariables": true,
+    "ifx": {},
+    "appsettings": {}
+}
+```
+This setting is primarily intended to ensure consistent environment configuration for testing purposes.
+
+## appsettings.json
+After loading environment variables (or not), the Config class loads the `appsettings.json` file.  `appsettings.json` is technically optional.  This file should never contain sensitive keys or secrets, as it is checked into source control.  Also, unintended values in this file could override environment variables in non-development environments.
```

Comment:
```suggestion
After loading environment variables (or not), the Config class loads the `appsettings.json` file, if included in the project. Values from appsettings.json will override values loaded from environment variables.

> [!CAUTION]
> appsettings.json should never contain sensitive keys or secrets, as this file is checked into source control.

> [!CAUTION]
> Because appsettings.json values will override matching environment variables, it's especially easy for checked-in appsettings.json values to unintentionally override environment variables in non-development environments.
```

Reply:
Note, I haven't used these things before but I think it'll render like this (if all goes well)

![image](https://github.com/user-attachments/assets/52b1179f-0d0c-4779-a986-3e6b7d611a7b)